posthog/api/action.py

```diff
@@ -176,12 +184,7 @@ def safely_get_queryset(self, queryset):
             queryset = queryset.filter(deleted=False)
 
         queryset = queryset.annotate(count=Count(TREND_FILTER_TYPE_EVENTS))
-        queryset = queryset.prefetch_related(Prefetch("steps", queryset=ActionStep.objects.order_by("id")))
+        queryset = queryset.prefetch_related(
+            "plugin_configs", Prefetch("steps", queryset=ActionStep.objects.order_by("id"))
+        )
         return queryset.filter(team_id=self.team_id).order_by(*self.ordering)
-
-    def list(self, request: request.Request, *args: Any, **kwargs: Any) -> Response:
```

Comment:
This was a weird override as it does the same as the built in viewset so removed it

Reply:
I'm reverting this change for now, as this has limited the taxonomic popover and the actions page to a maximum of 100 actions. Yes, we load them all and handle actions search frontend side 🤷‍♂️ Will try to move this backend side, but couldn't get it working quickly.